cmd/contour/bootstrap.go

```diff
@@ -37,6 +37,7 @@ func registerBootstrap(app *kingpin.Application) (*kingpin.CmdClause, *bootstrap
 	bootstrap.Flag("envoy-cert-file", "gRPC Client cert filename for Envoy to load").Envar("ENVOY_CERT_FILE").StringVar(&ctx.config.GrpcClientCert)
 	bootstrap.Flag("envoy-key-file", "gRPC Client key filename for Envoy to load").Envar("ENVOY_KEY_FILE").StringVar(&ctx.config.GrpcClientKey)
 	bootstrap.Flag("namespace", "The namespace the Envoy container will run in").Envar("CONTOUR_NAMESPACE").Default("projectcontour").StringVar(&ctx.config.Namespace)
+	bootstrap.Flag("enable-ads", "Enable the Envoy Aggregated Discovery Service").BoolVar(&ctx.config.EnableADS)
```

Comment:
Is there a reason to not just turn this on and not allow for a flag?

Reply:
Not especially, just risk mitigation. I'd be OK with removing it, or flipping the default. I'd slightly prefer the latter, since users could flip ADS off if something horrible happens :)

Reply:
Flipped this so it is enabled by default.